src/FileReader.java

```diff
@@ -0,0 +1,34 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+class FileReader {
+
+    /* Checking if the file is either txt or json  */
+
+    public static boolean isFileExists() {
+        try {
+            if (FileSearchApplication.filepath.endsWith("txt") || FileSearchApplication.filepath.endsWith("json")) {
```

Comment:
You should not be accessing filepath directly like this. You should accept this as a function parameter.

Reply:
Now I have used local variables and accessing through function parameters.